atlas/commands/bootstrap/bootstrap.go

```diff
@@ -136,10 +137,20 @@ func generateProtobuf() error {
 	return nil
 }
 
-// initDep calls "dep init" to generate .toml files
-func initDep() error {
-	fmt.Print("Starting dep project... ")
-	if err := runCommand("dep", "init"); err != nil {
+// initGoMod calls "go mod init" to generate .toml files
+func initGoMod() error {
+	fmt.Print("Starting mod project... ")
+	os.Setenv("GO111MODULE", "on")
+	if err := runCommand("go", "mod", "init"); err != nil {
+		fmt.Println(err)
```

Comment:
Error will be logged twice, see https://github.com/infobloxopen/atlas-cli/blob/928fd6164ad8819572c68279a30290da65fd7486/atlas/main.go#L24